modules/table/src/testFixtures/java/org/apache/ignite/distributed/ItTxTestCluster.java

```diff
@@ -940,11 +906,25 @@ public void shutdownCluster() {
             assertThat(clientTxManager.stopAsync(new ComponentContext()), willCompleteSuccessfully());
         }
 
-        // TODO: should be removed in https://issues.apache.org/jira/browse/IGNITE-22774
-        for (Map.Entry<String, List<RaftGroupService>> e : raftClients.entrySet()) {
-            for (RaftGroupService svc : e.getValue()) {
-                svc.shutdown();
+        Set<TablePartitionId> grpIds = tables.values()
+                .stream()
+                .map(table -> new TablePartitionId(table.tableId(), 0))
+                .collect(toSet());
+
+        for (String consistentId : clusterServices.keySet()) {
+            ReplicaManager replicaManager = replicaManagers.get(consistentId);
+
+            if (replicaManager == null) {
+                continue;
             }
+
+            grpIds.forEach(id -> {
+                try {
+                    replicaManager.stopReplica(id);
+                } catch (NodeStoppingException e) {
```

Comment:
Why do we need this no-op catch? I would prefer to see this exception and not sweep it under the carpet

Reply:
I agree, but it's a stopping process for tests -- I'm not sure what should I do in case `NSE` while a partial node is stopping?

Reply:
The code is deleted because it duplicates replica's stopping above the code